<launch>
    <node pkg="lab3" name="detect_lines" type="lab3.py" output="screen">
    </node>
    <node pkg="rqt_image_view" type="rqt_image_view" name="all_lines" args="/danietown/allLines"/>
    <node pkg="rqt_image_view" type="rqt_image_view" name="white_lines" args="/danietown/whiteLines"/>
    <node pkg="rqt_image_view" type="rqt_image_view" name="yellow_lines" args="/danietown/yellowLines"/>

    <!--Use below to start debug images-->
    <node pkg="image_processing_hw" type="image_pub_all.py" name="image_pub_all" output="screen">
    </node>
</launch>